pages/linux/fcrackzip.md

```diff
@@ -0,0 +1,27 @@
+# fcrackzip
+
+> ZIP archive password cracking.
+
+- Bruteforce password with length 4 to 8 with only lowercase, uppercase and digit characters (order matters):
```

Comment:
Perhaps this will read better?
```suggestion
- Brute-force a password with a length of 4 to 8 characters, and contains only alphanumeric characters (order matters):
```